packaging/packager_pip.sh

```diff
@@ -12,9 +12,11 @@ readonly release_tag="${RELEASE_TAG:-}"
 
 # disable tests and support of backends based on python versions
 if [[ $python_version == '3.9' ]]; then export DISABLE_AUTOPY=1; fi
-
```

Comment:
It seems to me that some empty lines separating the blocks have been removed here.

Reply:
How about this previous comment I had? It doesn't seem to be resolved.